web-common/src/components/banner/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const BillingBannerID = "banner";
+export const DashboardBannerID = "dashboard";
+export const TokenBannerID = "token";
+
+export const BillingBannerPriority = 1;
+export const DashboardBannerPriority = 2;
+export const TokenBannerPriority = 3;
```

Comment:
I think we should flip the priority of these. Then our pattern is 1. Rill Cloud concerns (billing, user permissions), 2. Dashboard concerns.